src/get_articles.rs

```diff
@@ -60,9 +60,34 @@ pub fn run(args: Args) -> anyhow::Result<()> {
         Default::default()
     };
 
-    if let Some(path) = args.osm_tags {
+    if let Some(ref path) = args.osm_tags {
         info!("Loading wikipedia/wikidata osm tags from {path:?}");
-        parse_osm_tag_file(path, &mut wikidata_qids, &mut wikipedia_titles)?;
+
+        let original_items = wikidata_qids.len() + wikipedia_titles.len();
+        let mut line_errors = Vec::new();
+        parse_osm_tag_file(
+            path,
+            &mut wikidata_qids,
+            &mut wikipedia_titles,
+            Some(&mut line_errors),
+        )?;
+
+        if !line_errors.is_empty() {
+            let error_count = line_errors.len();
+            let new_items = wikidata_qids.len() + wikipedia_titles.len() - original_items;
+            let expected_threshold = 0.02;
```

Comment:
What is the benefit of hiding errors under a threshold? Isn't it beneficial to see all errors and be able to estimate/compare the quality of the dump, and to easily grep/find what is most important, or feed the whole log to contributors for fixes?

Reply:
The threshold only determines if the message is `info` vs `error` level.
When you use the `run.sh` script with multiple languages it prints a copy of the hundreds of errors for each language.
I think writing the parse errors to a file separately will be easier to read and deal with.

I'm open to other ideas.